File: test/golden/java/neoforge/src/main/java/com/example/testmod/neoforge/TestmodModNeoForge.java
package com.example.testmod.neoforge;

import com.example.testmod.TestmodMod;
import net.neoforged.bus.api.IEventBus;
import net.neoforged.fml.common.Mod;

@Mod(TestmodMod.MOD_ID)
public class TestmodModNeoForge {
    public TestmodModNeoForge(IEventBus modEventBus) {
        TestmodMod.init();
    }
}

File: test/golden/kotlin/neoforge/src/main/kotlin/com/example/testmod/neoforge/TestmodModNeoForge.kt
package com.example.testmod.neoforge

import com.example.testmod.TestmodMod
import net.neoforged.bus.api.IEventBus
import net.neoforged.fml.common.Mod

@Mod(TestmodMod.MOD_ID)
class TestmodModNeoForge(modEventBus: IEventBus) {
    init {
        TestmodMod.init()
    }
}
